.mage/dev.go

```diff
@@ -85,6 +90,83 @@ func execDockerCompose(args ...string) error {
 	return err
 }
 
+func execDockerComposeWithOutput(filepath string, args ...string) error {
+	output, err := sh.Output("docker-compose", dockerComposeFlags(args...)...)
+	if err != nil {
+		return err
+	}
+	message := []byte(output)
+	err = ioutil.WriteFile(filepath, message, 0644)
+	return err
+}
+
+func node() (func(args ...string) error, error) {
```

Comment:
Is this function really necessary?

Reply:
Yeah, so instead of redefining it here, we can make the function in `js.go` global.

Reply:
Please just use `sh.Run("node")` instead, there's no reason to have a helper function for this.